Sector Ownership/Non-UK/France - Lower APP.txt

```diff
@@ -0,0 +1,48 @@
+;Lille (LFQQ)
+
+SECTOR:LFQQ:0:0
+OWNER:QQR:PTB:TNTB:AENB:RPAW:P
+
+SECTOR:LFFF QD QW (DB-085):0:8500
+OWNER:QQR:PTB:TNTB:AENB:RPAW:P
+BORDER:LSFF:LDM16:LDM01:LDM02:LDFF03:LDFF02:LDFF01:LFFF_EBBU:LFQQ_southline
+
+;Iroise (LFRB)
+
+SECTOR:LFRB:0:0
+OWNER:RBR:BR
+
+SECTOR:LFRB 6.1L (DB-115):11500:19500
+OWNER:RBIR:RBR:BR
+BORDER:LW06A:LW06B:LFRB_LW:JJCTR-N-SKERY-LFRR:JJCTR-M-SKERY-LFRR:JJCTR-S-SKERY-N:LFRB_SIV5_SIV6:LFRB-SIV-6.1U_southline:LFRB_LFRR
```

Comment:
Is this border supposed to use the 6.1U? Just given the sector name I wondered if it should be an L version? Or is it just used for both?